packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/getAndroidSplashConfig.ts

```diff
@@ -13,6 +13,7 @@ export type SplashScreenConfig = {
 export type AndroidPluginConfig = {
   backgroundColor: string;
   logoWidth: number;
+  image?: string | null;
```

Comment:
I would use `logo` (to fit with `logoWidth`, and as `brand` might come after)

Reply:
We're trying to keep the config backwards compatible. Trying to make it so the only necessary change is have to call `hide`

Reply:
Maybe it would be better to take a new prop (since the image is going to be different anyways, no?) and use the app icon as a fallback with a warning instead?